### ISSUE TITLE
Repairs: "The unit of ... has changed"

### ISSUE THREAD

**Issue by WendelFelius** [jcgoette/baby_buddy_homeassistant#153]
After installing Home Assistant Core 2024.10 it offered some repairs for Baby Buddy entities. This is the one for the lastfeeding-sensor:  

> The unit of 'xxx last feeding' (**sensor.xxx_last_feeding**) changed to '' which can't be converted to the previously stored unit, '**mL**'. If the historic statistic values have a wrong unit, you can update the units of the old values. The values will not be updated. Otherwise you can choose to delete all historic statistic values, and start over.
> 
> How do you want to fix this issue?
> [X] Update the unit of the historic statistic values from 'mL' to '', without converting.
> [O] Delete all old statistic data for this entity

Also getting them for _last temperature (°C)_ and _last weight (kg)_. 

Should I choose a fix myself (if so, which?) or wait for an update of the baby buddy integration?

**Comment by jcgoette** [comment]
I don't think this has anything to do with the integration per se. I personally had it come up with an unrelated sensor also.

**Comment by WendelFelius** [comment]
```
Logger: homeassistant.components.sensor
Source: components/sensor/__init__.py:731
integration: Sensor (documentation, issues)
First occurred: 12:05:14 (1 occurrences)
Last logged: 12:05:14

Entity sensor.babyname_last_temperature (<class 'custom_components.babybuddy.sensor.BabyBuddyChildDataSensor'>) is using native unit of measurement 'None' which is not a valid unit for the device class ('temperature') it is using; expected one of ['°F', 'K', '°C']; Please update your configuration if your entity is manually configured, otherwise create a bug report at https://github.com/jcgoette/baby_buddy_homeassistant/issues
```
